Swift_Alert/Swift_Alert/ContentView.swift

```diff
@@ -8,14 +8,104 @@
 import SwiftUI
 
 struct ContentView: View {
+    
+    private enum Constants {
+        static let showTitle = "Показать"
+        static let firstBlockTitle = "Пример Alert"
+        static let secondBlockTitle = "Пример Alert c 2 кнопками и логикой"
+        static let secondBlockTitleAfterTap = "Другой текст"
+        static let thirdBlockTitle = "Пример ActionSheet"
+        static let fourthBlockTitle = "Пример ActionSheet c кнопками и логикой"
+        static let fourthBlockTitleAfterTap = "Пример ActionSheet c кнопками и логикой"
+        static let changeTitle = "Изменить текст"
+        static let changeColorTitle = "Сменить цвет?"
+        static let transferMoneyTitle = "Перевести деньги"
+        static let yesTitle = "Да"
+        static let cancelTitle = "Отмена"
+        static let areYouSureTitle = "Вы уверены?"
+        static let answerTitle = "Выберите вариант ответа"
+        static let spacing: CGFloat = 25
+
+    }
+    
+    @State var testFirstView = false
+    @State var testSecondView = false
+    @State var forFirstAlertFlag = false
+    @State var forSecondAlertFlag = false
+    @State var testActionSheet = false
+    @State var testActionSheetWithButton = false
```

Comment:
```suggestion
    @State private var isFirstViewShown = false
    @State private var isSecondViewShown = false
    @State private var isFirstAlertShown = false
    @State private var isSecondAlertShown = false
    @State private var isActionSheetShown = false
    @State private var isActionSheetWithButtonShown = false
```
 где флаг что-то показывает, то в начале пишем is  в конце  Shown